pkg/tlog/entry.go

```diff
@@ -66,21 +67,24 @@ func NewEntry(body []byte, integratedTime int64, logIndex int64, logID []byte, s
 	if err != nil {
 		return nil, err
 	}
-	rekorEntry, err := types.UnmarshalEntry(pe)
```

Comment:
One downside to removing this is there's certain correctness checks that occur during unmarshalling that will now be skipped.